app/client/sagas/creditTransferSagas.js

```diff
@@ -27,10 +27,11 @@ import {handleError} from "../utils";
 
 function* updateStateFromCreditTransfer(result) {
   //Schema expects a list of credit transfer objects
+  let credit_transfer_list = [];
   if (result.data.credit_transfers) {
-    var credit_transfer_list = result.data.credit_transfers
```

Comment:
this is dangerous code, could end up with unexpected global variables. always declare a `var` (ES6 `const` or `let`) top of scope before using it in lower scopes

Reply:
thanks, nice catch